src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```
@@ -40,6 +40,8 @@ Custom theme
 /******************************************
 Datatable
 ******************************************/
+@include type.type-classes();
+$purple-60-dark: #7f3ae7;
```

colors should be added inside the `_bootstrap-defaults.scss` file since we are still using those colors across. 

and maybe name it in a way that its clear we are using this color for this particular usecase
```suggestion
$inline-tip-bg-color: #7f3ae7;
```

---

Will we deprecate bootstrap-defaults? even if it says bootstrap-defaults, its actually the ceph-dashboard color pattern of the upstream which we will continue to use even if carbonize our entire application.. so maybe the most we would do there is to just rename the file?

---

No, not really from there, the carbon has color palette. But, okay to do later.